Cloud_Community_REST_API_Docs.md

```
@@ -0,0 +1,1065 @@
+        "details": {}
+    }
+}
+```
+
+---
+
+### 댓글 작성
+
+POST `/posts/comments`
+
+게시글에 새로운 댓글을 작성합니다. 댓글을 작성하려면 로그인이 필수적으로 이루어져야 합니다. 댓글 작성에 성공하면 자동으로 생성시간이 할당됩니다. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)